addons/trenches/functions/fnc_applyCamouflageAttribute.sqf

```diff
@@ -0,0 +1,25 @@
+#include "script_component.hpp"
+/*
+ * Author:  Christian 'chris5790' Klemm
+ * Apply camouflage attribute to trench.
+ *
+ * Arguments:
+ * 0: object <OBJECT>
+ * 1: value <INT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [TrenchObj, ACE_player] call ace_trenches_fnc_applyCamouflageAttribute
+ *
+ * Public: No
+ */
+
+params ["_object", "_value"];
+
+if (_value isEqualTo 1) then {
+    [_object] call FUNC(placeCamouflage);
+} else {
+    [_object] call FUNC(deleteCamouflage);
+};
```

Comment:
Missing empty newline at EOF. Won't comment further for the other files.

Reply:
There is an empty newline at EOF on my side, github just does not carry it over